lib/fizzy/types.hpp

```diff
@@ -22,6 +22,8 @@ enum class ValType : uint8_t
 /// https://webassembly.github.io/spec/core/binary/types.html#result-types
 constexpr uint8_t BlockTypeEmpty = 0x40;
 
+constexpr uint8_t FuncRef = 0x70;
```

Comment:
Maybe add link to https://webassembly.github.io/spec/core/binary/types.html#table-types